lib/react_client.dart

```diff
@@ -225,9 +250,15 @@ final ReactDartInteropStatics _dartInteropStatics = (() {
   /// 1. Update [Component.props] using the value stored to [Component.nextProps]
   ///    in `componentWillReceiveProps`.
   /// 2. Update [Component.state] by calling [Component.transferComponentState]
-  void _afterPropsChange(Component component) {
+  /// 3. Update [Component.context] by calling [Component.transferComponentContext]
+  void _afterPropsChange(Component component, InteropContextValue nextContext) {
     component.props = component.nextProps; // [1]
     component.transferComponentState();    // [2]
+    component.transferComponentContext();  // [3]
```

Comment:
From this code, and from testing the example, context doesn't seem to ever get updated. 

`nextState` gets updated in `setState` and then transferred to `state` in `transferComponentContext`, but there doesn't seem to be an analagous updating of `nextContext` here.

I would recommend updating how context is managed to mirror `props` as opposed to `state`, since they both are updated externally.

This updating should also have test coverage (see later comment in tests).

Reply:
I believe that is a react-js issue: https://github.com/facebook/react/issues/2517.

Reply:
I believe that issue is around how to handle context not getting updated when `shouldComponentUpdate` returns `false`. 

Otherwise, context should update.

Reply:
I agree. I'm validating that. The initial code had this issue and I found several references of it not updating.

I'll ping back once I confirm it.

Reply:
Built a react-js app and it does indeed update. Investigating it.

Reply:
Ok...it is refactored to match the props integration.